crates/astria-bridge-withdrawer/src/config.rs

```diff
@@ -26,6 +26,8 @@ pub struct Config {
     pub rollup_asset_denomination: asset::denom::TracePrefixed,
     // The bridge address corresponding to the bridged rollup asset on the sequencer.
     pub sequencer_bridge_address: String,
+    // The timeout duration for `Ics20Withdrawal`s in seconds.
+    pub ics20_withdrawal_timeout_duration: u64,
```

Comment:
Likewise, since this isn't a `Duration` yet, renaming to `ics20_withdrawal_timeout_seconds` might be slightly clearer?